app/src/main/java/com/cmput301f20t21/bookfriends/datatransfer/BookDataTransferHelper.java

```diff
@@ -16,7 +16,7 @@
 import com.google.gson.GsonBuilder;
 
 // https://stackoverflow.com/a/18463758
-public class BookDataTransferHelper {
+    public class BookDataTransferHelper {
```

Comment:
I think we no longer need anything is in package, you want to just delete them as part of this PR?